features/old/menu/dashboard.feature

```diff
@@ -1,10 +1,10 @@
 Feature: Dashboard
   In order to navigate easily
-  As a provider
+  As a provider with backend api
   I want to have important links on the dashboard
 
   Background:
-    Given a provider "foo.example.com"
+    Given a provider "foo.example.com" with backend api
```

Comment:
"Given a provider" implies "with a service". Why wouldn't it imply "with backend api"? I think we should define tests in the other way around: "Given a provider **without** backend api" and avoid changing a bunch of cukes.